platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/util/PropertiesFileSupport.java

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
I'm not particularly concerned with mocking and it shouldn't affect the correctness of our code.